crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/lambda.py

```diff
@@ -203,3 +203,28 @@ def f(
     y:
     z
 )
+
+lambda self, araa, kkkwargs=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa(*args, **kwargs), e=1, f=2, g=2: d
+
+# Regression tests for https://github.com/astral-sh/ruff/issues/8179
```

Comment:
This PR does not fix this formatting. It only adds tests for it

Reply:
What _does_ this PR fix? Sorry, trying to parse!

Reply:
Wait sorry, I now understand.